src/main/java/org/folio/dew/batch/acquisitions/edifact/services/ConfigurationService.java

```diff
@@ -1,35 +1,47 @@
 package org.folio.dew.batch.acquisitions.edifact.services;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import org.folio.dew.client.ConfigurationClient;
-import org.folio.dew.domain.dto.ConfigurationCollection;
-import org.springframework.stereotype.Service;
+import java.util.UUID;
 
 import lombok.RequiredArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dew.client.ConfigurationClient;
+import org.folio.dew.domain.dto.ModelConfiguration;
+import org.springframework.boot.configurationprocessor.json.JSONException;
+import org.springframework.boot.configurationprocessor.json.JSONObject;
```

Comment:
This dependency is usually used in development, the final artefact/jar that will be running in production may omit it in its build and result in a Runtime Exception, such as this: https://stackoverflow.com/questions/73071773/how-to-fix-java-lang-classnotfoundexception-org-springframework-boot-configurat